server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -63,13 +64,15 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
 	constructor(
 		private readonly mongoManager: MongoManager,
 		private readonly documentCollection: ICollection<IDocument>,
+        private readonly localCheckpointCollection: ICollection<ICheckpoint>,
```

Comment:
Nit, here should be tab. Please install `prettier` extension and set it to read from config. Details https://teams.microsoft.com/l/message/19:fa0021b5a71849eb9d46f963e7d231f0@thread.tacv2/1678385882120?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=97c01198-1d9b-4f3e-aa4d-bd7adf9430e6&parentMessageId=1678385882120&teamName=Fluid%20Relay%20Service&channelName=FRS%20Dev&createdTime=1678385882120